options/locale/locale_en-US.ini

```diff
@@ -1505,6 +1505,40 @@ milestones.filter_sort.most_complete = Most complete
 milestones.filter_sort.most_issues = Most issues
 milestones.filter_sort.least_issues = Least issues
 
+packages = Packages
+packages.desc = Manage repository packages.
+packages.filter.package_type = Type
+packages.filter.package_type.all = All
+packages.published_by = Published %[1]s by <a href="%[2]s">%[3]s</a>
+packages.installation = Installation
+packages.about = About this package
+packages.dependencies = Dependencies
+packages.details = Details
+packages.details.author = Author
+packages.details.project_site = Project Site
+packages.details.license = License
+packages.assets = Assets
+packages.versions = Versions
+packages.versions.on = on
+packages.delete = Delete package
+packages.delete.notice = You are about to delete %s (%s). This operation is irreversible, are you sure?
+packages.delete.success = The package has been deleted.
+packages.generic.use = Download the package, for example with curl:
+packages.generic.documentation = For more information on the generic registry, see <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/generic">the documentation</a>.
+packages.maven.registry = Setup this registry in your projects <code>pom.xml</code> file:
+packages.maven.use = To use the package include the following in the <code>dependencies</code> block in the <code>pom.xml</code> file:
+packages.maven.use_2 = Alternatively use this Maven command:
+packages.maven.documentation = For more information on the Maven registry, see <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/maven/">the documentation</a>.
+packages.nuget.registry = Setup this registry with the following command:
+packages.nuget.use = Install the package with the following command:
+packages.nuget.documentation = For more information on the NuGet registry, see <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/nuget/">the documentation</a>.
+packages.npm.registry = Setup this registry in your projects <code>.npmrc</code> file:
+packages.npm.use = To install the package use the following command:
+packages.npm.documentation = For more information on the npm registry, see <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/npm/">the documentation</a>.
+packages.pypi.requires = Requires Python
+packages.pypi.use = To install the package use the following command:
```

Comment:
I am not sure, but I would say that a comma belongs here.
If that is indeed the case, the same applies to `packages.nuget.use` and `packages.npm.use`.
```ini
packages.pypi.use = To install the package, use the following command:
```
<br>

Additionally, `packages.nuget.use`, `packages.npm.use` and `packages.pypi.use` could potentially all be merged into one string (maybe called `packages.installation.command`) as they all have the exact same meaning (but differ slightly in the wording for NuGet)